client/ayon_core/tools/pyblish_pype/model.py

```diff
@@ -780,6 +780,8 @@ def restore_checkstates(self):
 
     def update_with_result(self, result):
         instance = result["instance"]
+        if isinstance(instance, list):
```

Comment:
Wait a minut, `Context` and `Instance` are both `list`.

Reply:
Reverted fix, it may cause havoc.
I'll try switch Harmony to Publisher UI instead.